packages/orchestration/src/exos/icq-connection-kit.js

```diff
@@ -89,20 +85,17 @@ export const prepareICQConnectionKit = (zone, { watch, when }) =>
         },
         /**
          * @param {JsonSafe<RequestQuery>[]} msgs
-         * @returns {Promise<JsonSafe<ResponseQuery>[]>}
-         * @throws {Error} if packet fails to send or an error is returned
```

Comment:
too badd there's no `@rejects` or whatever. Shouldn't this doc still inform the user of that?